examples/Example/src/new_api/camera/index.tsx

```diff
@@ -0,0 +1,351 @@
+    const selectedFilter = Math.round(
+      Math.min(filters.length - 1, Math.max(filter.value, 0))
+    );
+    setSelectedFilter(selectedFilter);
+
+    return selectedFilter;
+  }
+
+  function takePhoto() {
+    Alert.alert('You took a photo');
+  }
+
+  function takeSeries() {
+    Alert.alert('You took a series of photos');
+  }
+
+  function startRecording() {
+    setIsRecording(true);
+    setRemainingTime(MAX_VIDEO_DURATION);
+    setRecordingIntervalHandle(
+      setInterval(() => {
+        setRemainingTime((r) => r - 200);
+      }, 200)
+    );
+  }
+
+  function finishRecording() {
+    setIsRecording(false);
+    clearInterval(recordingIntervalHandle);
+    setRemainingTime(MAX_VIDEO_DURATION);
+
+    Alert.alert(
+      `You took a video (${(MAX_VIDEO_DURATION - remainingTime) * 0.001} s)`
+    );
+  }
+
+  const zoomStyle = useAnimatedStyle(() => {
+    return { transform: [{ scale: zoom.value }] };
+  });
+
+  return (
+    <Animated.View style={styles.container}>
+      <GestureMonitor gesture={scaleGestureHandler}>
+        <Animated.View
+          style={[styles.home, { backgroundColor: filters[selectedFilter] }]}>
+          <Animated.View style={[styles.box, zoomStyle]} />
+        </Animated.View>
+      </GestureMonitor>
+
+      <GestureMonitor gesture={panGestureHandler}>
+        <Animated.View style={styles.buttonContainer}>
+          <FilterCarousel filters={filters} selected={filter} />
+          <GestureMonitor gesture={buttonGestureHandler}>
+            <CaptureButton
+              progress={1 - remainingTime / MAX_VIDEO_DURATION}
+              onTimerFinished={() => {
+                finishRecording();
+              }}
+            />
+          </GestureMonitor>
+        </Animated.View>
+      </GestureMonitor>
+    </Animated.View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    width: '100%',
+    height: '100%',
+    alignSelf: 'center',
+    backgroundColor: 'white',
+  },
+  home: {
+    width: '100%',
+    height: '100%',
+    alignSelf: 'center',
+    opacity: 0.5,
+  },
+  buttonContainer: {
+    width: '100%',
+    height: CAPTURE_BUTTON_RADIUS * 2,
+    position: 'absolute',
+    bottom: 50,
+    zIndex: 10,
+  },
+  box: {
+    alignSelf: 'center',
+    margin: 100,
+    width: 200,
+    height: 200,
+    backgroundColor: 'black',
+  },
+});
+
+function FilterCarousel(props: {
+  filters: string[];
+  selected: Animated.SharedValue<number>;
+}) {
+  return (
+    <Animated.View style={filterCarouselStyles.container}>
+      {props.filters.map((filter) => (
+        <Filter key={filter} filter={filter} selected={props.selected} />
+      ))}
+    </Animated.View>
+  );
+}
+
+function Filter(props: {
+  filter: string;
+  selected: Animated.SharedValue<number>;
+}) {
+  const window = useWindowDimensions();
+  const style = useAnimatedStyle(() => {
+    return {
+      transform: [
+        {
+          translateX:
+            window.width * 0.5 -
```

Comment:
```suggestion
            window.width / 2 -
```

Reply:
Done in c9c43d8757b585808251376d16f6a0182c2842f3